schemas/clinvar-study.tables.json

```diff
@@ -0,0 +1,3068 @@
+            "array_of": true
+          },
+          {
+            "name": "xref",
+            "datatype": "string",
+            "array_of": true
+          }
+        ]
+      },
+      {
+        "name": "sequence_location",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "for_display",
+            "datatype": "boolean"
+          },
+          {
+            "name": "asssembly",
+            "datatype": "string"
+          },
+          {
+            "name": "chr",
+            "datatype": "string"
+          },
+          {
+            "name": "accession",
+            "datatype": "string"
+          },
+          {
+            "name": "outer_start",
+            "datatype": "integer"
+          },
+          {
+            "name": "inner_start",
+            "datatype": "integer"
+          },
+          {
+            "name": "start",
+            "datatype": "integer"
+          },
+          {
+            "name": "stop",
+            "datatype": "integer"
+          },
+          {
+            "name": "inner_stop",
+            "datatype": "integer"
+          },
+          {
+            "name": "outer_stop",
+            "datatype": "integer"
+          },
+          {
+            "name": "display_start",
+            "datatype": "integer"
+          },
+          {
+            "name": "display_stop",
+            "datatype": "integer"
+          },
+          {
+            "name": "strand",
+            "datatype": "string"
+          },
+          {
+            "name": "variant_length",
+            "datatype": "integer"
+          },
+          {
+            "name": "reference_allele",
+            "datatype": "string"
+          },
+          {
+            "name": "alternate_allele",
+            "datatype": "string"
+          },
+          {
+            "name": "assembly_acession_version",
+            "datatype": "string"
+          },
+          {
+            "name": "assembly_status",
+            "datatype": "string"
+          },
+          {
+            "name": "position_vcf",
+            "datatype": "integer"
+          },
+          {
+            "name": "reference_allele_vcf",
+            "datatype": "string"
+          },
+          {
+            "name": "alternate_allele_vcf",
+            "datatype": "string"
+          },
+          {
+            "name": "for_display_length",
+            "datatype": "boolean"
+          }
+        ]
+      },
+      {
+        "name": "type_chromosome",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "type_chromosome_nr",
+            "datatype": "string"
+          },
+          {
+            "name": "type_chromosome_str",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_chromosome_nr",
```

Comment:
This table feels unneeded

Reply:
Agree. I think we can eliminate `type_chromosome`, `type_chromosome_nr` and `type_chromosome_str` as tables and just put the chromosome value into `sequence_location.chr`